METADATA/README.md

```diff
@@ -321,3 +322,46 @@ List of all character subsets available in Google Fonts API for the given font f
 * `thai`
 * `tibetan`
 * `vietnamese`
+
+### `axes`
+
+Repeated for each axis to be available in the Google Fonts API for the given family.
+
+Contains 3 required child values: 
+
+* 1st, `tag`, with a string value for the 4 character axis tag code.
+* `min_value` and `max_value` as `int` or `float` values
+
+
+Example:
+
+```
+axes {
+  tag: "wght"
+  min_value: 300.0
+  max_value: 700.0
+}
+axes {
+  tag: "opsz"
+  min_value: 10
+  max_value: 144
+}
+```
+
+### `registry_default_overrides`
```

Comment:
Can you also mention that this is required for an axis if it's [min, max] do not contain the registry default.

Reply:
@garretrieger what does that look like in code? eg

```
registry_default_overrides {
  key: "opsz"
  value: 18
  min: 18
  max: 24
}
```

Reply:
Close, just drop the min/max:

registry_default_overrides {
  key: "opsz"
  value: 18
}

The default override only overrides the default value.